debugging/coreclr/dockerClient.ts

```diff
@@ -187,12 +200,14 @@ export class CliDockerClient implements DockerClient {
         options = options || {};
 
         const command = CommandLineBuilder
-            .create('docker', 'run', '-dt', '-P')
+            .create('docker', 'run', '-dt', options.ports ? '-P' : '')
```

Comment:
`options.ports` will be true even if empty; you'll need to test both for validity as well as length > 0.

Ideally this would be replaced with a new function in `CommandLineBuilder` that represents flags, something like:

`.withFlagArg('-P', options.ports === undefined || options.ports.length < 1)`

Reply:
Thanks for pointing out the `.withFlagsArg()` method, I'll use it and remove the `-P` flag from the `.create()`.